CHANGELOG.md

```diff
@@ -14,13 +14,17 @@
 
 - Arbitrary query parameters now passed along to authorize URL creation
 - Fixed build issue when using Windows PowerShell
-- WebAuthn error handling is now standardized according to the WebAuthn spec
-- When WebAuthn encounters an error, it formats it appropriately and injects it into the hiddenCallback, allowing the developer to just send to AM or handle it specially (error is still thrown)
+- WebAuthn error handling is not standardized according to the WebAuthn spec
+- When WebAuthn encounters an error, it formats is appropriately and injects it into the hiddenCallback, allowing the developer to just send to AM or handle it specially.
```

Comment:
It looks like `(error is still thrown)` got removed from this item.

Reply:
fixed